articles/native-platforms/ios-swift.md

```diff
@@ -177,6 +177,10 @@ Then add Lock Facebook's Pod
 pod 'Lock-Facebook', '~> 2.0'
 ```
 
+Click on the `Pods` project and select `Lock-Facebook` target and set `Allow Non-modular Includes In Framework Modules` to Yes:
```

Comment:
I've tried to add Bolts dependency but it didn't help. Shouldn't it be done within Lock-Facebook?
By the way they also proposed to set this property to YES:
![image](https://cloud.githubusercontent.com/assets/320958/9604023/6db45a02-50bd-11e5-982d-42f7863f12cc.png)

Finally, should we remove this setting from the tutorial?